bin/run_hpc_at_cms.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# This script set up the environment and run the hpc_at_cms.py
+# Run with --help to see usage instructions. 
+# With this environment, It works in lxplus7, it can requiere modifications to run elsewhere.
+# See https://its.cern.ch/jira/browse/CMSMONIT-341 for mor details.
+
+source /cvmfs/sft.cern.ch/lcg/views/LCG_96python3/x86_64-centos7-gcc8-opt/setup.sh
+source /cvmfs/sft.cern.ch/lcg/etc/hadoop-confext/hadoop-swan-setconf.sh analytix
+currentDir=$(
+  cd "$(dirname "$0")" && pwd
+)
+# In lxplus, when running with acrontab, we need to set the java home
+# to a jvm with avanced encryption enabled. 
+# see https://cern.service-now.com/service-portal/view-request.do?n=RQF1380598 
+
+if [ -e "/usr/lib/jvm/java-1.8.0" ]
```

Comment:
here you should first check if JAVA_HOME is set in environment and use it. This will allows to move smoothly to next java version